src/targets/gpu/include/migraphx/gpu/convolution.hpp

```diff
@@ -17,19 +17,25 @@ struct miopen_convolution
     shared<convolution_descriptor> cd;
     miopenConvFwdAlgorithm_t algo{};
     miopenHandle_t handle = nullptr;
+    uint64_t solution_id  = 0;
```

Comment:
We can remove `algo` here since we have the solution_id.

Reply:
we still use `algo` for checking fusions when it's using Winograd